packages/usa-input-prefix-suffix/src/styles/_usa-input-prefix-suffix.scss

```diff
@@ -38,9 +38,24 @@ $icon-offset: $icon-width + ($input-padding * 2);
     min-width: 0;
     width: 100%;
 
-    &:disabled + .usa-input-suffix,
-    &[aria-disabled="true"] + .usa-input-suffix {
-      @include u-disabled;
+    &:disabled,
+    &[aria-disabled="true"] {
+      & + .usa-input-suffix {
+        @include u-disabled;
+      }
+
+      @media (forced-colors: active) {
+        // Override border from disabled input
+        border: 0;
```

Comment:
Could this be moved to the `u-disabled` mixin used above so it doesn't cause confusion with the duplicate media query? If so, we could also re-use on `_usa-input-mask.scss`.

https://github.com/uswds/uswds/blob/7413aeafca39fc45565de472a52964f025fbd043/packages/uswds-core/src/styles/mixins/utilities/_disabled.scss#L23-L33

Reply:
Resolved in [d746e06](https://github.com/uswds/uswds/pull/5295/commits/d746e06e784cce26b65243b07a35dd57feb4cc7a)

Had to add the border back for range slider, but easy to do with the disabled HC border mixin

```scss
  &::-webkit-slider-runnable-track {
    @include u-disabled;
    @include u-disabled-high-contrast-border;
  }

  &::-moz-range-track {
    @include u-disabled;
    @include u-disabled-high-contrast-border;
  }
```